src/core/primitives.rs

```diff
@@ -192,11 +194,13 @@ pub fn setup(
             s2,
             subkeys: sub_sk,
             kmac_key,
+            policy: policy.clone()
         },
         MasterPublicKey {
             g1,
             g2,
             subkeys: sub_pk,
+            policy : policy.clone()
```

Comment:
I don't know how to resolve this error : 
```
error[E0382]: use of moved value: `policy`
   --> src/core/primitives.rs:202:13
    |
177 |     let policy = Policy::new();
    |         ------ move occurs because `policy` has type `PolicyV2`, which does not implement the `Copy` trait
...
196 |             policy,
    |             ------ value moved here
...
202 |             policy,
    |             ^^^^^^ value used here after move
    |
help: consider cloning the value if the performance cost is acceptable
    |
196 |             policy: policy.clone(),
    |                   ++++++++++++++++
```
Could you give me a hint please ?